x-pack/plugins/upgrade_assistant/public/application/components/es_deprecation_logs/fix_deprecation_logs/deprecations_count_checkpoint/deprecations_count_checkpoint.tsx

```diff
@@ -96,16 +94,6 @@ export const DeprecationsCountCheckpoint: FunctionComponent<Props> = ({
     setCheckpoint(now);
   };
 
-  useEffect(() => {
-    // Loading shouldn't invalidate the previous state.
-    if (!isLoading) {
-      // An error should invalidate the previous state.
-      setHasNoDeprecationLogs(!error && !hasLogs);
-    }
-    // Depending upon setHasNoDeprecationLogs would create an infinite loop.
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [error, isLoading, hasLogs]);
```

Comment:
This was part of the logic used used for setting the step completion when there was no deprecation logs.